example_docs/docs/sphinx_guide/paragraph.rst

```diff
@@ -325,8 +325,7 @@ Admonitions
 .. Hint:: It's bigger than a bread box.
 
 .. Note:: This is a note.
-   Equations within a note:
-   :math:`G_{\mu\nu} = 8 \pi G (T_{\mu\nu}  + \rho_\Lambda g_{\mu\nu})`.
+   Equations within a note: what's up!
```

Comment:
was this change intentional? I think it would be good to keep some latex equation examples as they're used a lot by our docs

Reply:
Yeah, math is flaky with Playwright tests :( it renders slightly differently sometimes